client/src/components/Login.js

```diff
@@ -259,7 +264,8 @@ const Login = (props) => {
             fullWidth
             variant="contained"
             color="primary"
-            disabled={loading}
+            //Disable button if sending request, username or password are empty
+            disabled={loading || (!userName && usernameBlurred) || (!password && passwordBlurred)}
```

Comment:
We could simplify this a little and just disable the button if either `userName` or `password` is empty (without checking `blurred`.
I think that would make the user experience cleaner.

Reply:
So log in button state will be disabled when page is loaded, right
![Screenshot 2020-09-07 at 10 58 04](https://user-images.githubusercontent.com/15067440/92363007-7e9d9180-f0f9-11ea-9c91-5f691cf3374c.png)